app/serializers/api/v1/result_table_serializer.rb

```diff
@@ -10,6 +10,10 @@ def table_id
         object.table&.id
       end
 
+      def metadata
```

Comment:
should be called `table_metadata`, to keep it consistent with the other serialized table properties (table_id, table_contents)